packages/gatsby-source-nacelle/src/type-defs/index.js

```diff
@@ -1,11 +1,133 @@
 module.exports = `
```

Comment:
Were you not able to figure out a programmatic way to handle these updates to the schema? Just curious

Reply:
I spent some time trying to solve this via remote schema introspection, but didn't get very far. It's worth revisiting later, for sure.

Reply:
👍  We can resolve this, but should create a story to come back and take care of this.